### PR TITLE
tweak the expressInstall quickstart

### PR DESCRIPTION
**Pull request by qrkourier**
I've transplanted the changes from https://github.com/openziti/ziti/pull/1688 that were not strictly necessary for that branch to pass tests.

### CODE REVIEW

File: quickstart/docker/image/run-controller.sh [hunk 1]
```diff
@@ -60,4 +60,4 @@ unset ZITI_PWD
 # create a place for the internal db
 mkdir -p $ZITI_HOME/db
 
-"${ZITI_BIN_DIR}/ziti" controller run "${ZITI_HOME}/${ZITI_CTRL_NAME}.yaml"
```
Comment (qrkourier) [inline quickstart/docker/image/run-controller.sh:63]: add option to enable debug in the controller container log

File: quickstart/docker/image/run-router.sh [hunk 2]
```diff
@@ -72,5 +72,5 @@ fi
 unset ZITI_USER
 unset ZITI_PWD
 
-"${ZITI_BIN_DIR}/ziti" router run "${ZITI_HOME}/${ZITI_ROUTER_NAME}.yaml" > "${ZITI_HOME}/${ZITI_ROUTER_NAME}.log"
+"${ZITI_BIN_DIR}/ziti" router run "${ZITI_HOME}/${ZITI_ROUTER_NAME}.yaml"
```
Comment (qrkourier) [inline quickstart/docker/image/run-router.sh:75]: this was splitting the log and sending the stdout to a file and stderr to the container log, but containers should send both to the container log so they appear in Docker's built-in log facility

File: quickstart/docker/simplified-docker-compose.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ services:
   ziti-controller:
     image: "${ZITI_IMAGE}:${ZITI_VERSION}"
     healthcheck:
-      test: curl -m 1 -s -k https://${ZITI_CTRL_EDGE_ADVERTISED_ADDRESS:-ziti-edge-controller}:${ZITI_CTRL_EDGE_ADVERTISED_PORT:-1280}/edge/client/v1/version
+      test: curl -m 1 -s -k -f https://${ZITI_CTRL_EDGE_ADVERTISED_ADDRESS:-ziti-edge-controller}:${ZITI_CTRL_EDGE_ADVERTISED_PORT:-1280}/edge/client/v1/version
```
Comment (qrkourier) [inline quickstart/docker/simplified-docker-compose.yml:5]: the `-f` (`--fail`) option is necessary to raise an exception when the HTTP response is >=400

File: quickstart/docker/docker-compose.yml [hunk 4]
```diff
@@ -1,4 +1,3 @@
-version: '2.4'
```
Comment (dovholuknf) [inline quickstart/docker/docker-compose.yml:1]: is there a good reason to remove this?

Comment (qrkourier) [reply quickstart/docker/docker-compose.yml:1]: It's an obsolete property

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**